src/app/Library/CrudPanel/Traits/Query.php

```diff
@@ -29,12 +29,35 @@ trait Query
      *     $query->activePosts();
      * });
      *
-     * @param  callable|string  $function
-     * @return mixed
+     * @param  \Closure|string  $function
+     * @return void
```

Comment:
🚩 I see we're changing the return type of a `public` method here? Weee... can't really do that, can we? What was _actually_ returned before? Was it the query, so you could chain it? Then we DEFINITELY can't do this...

We might be forced to add this closure functionality later, in v6, as a breaking change...

Actually I'd be more comfortable to not change this at this point. We can then add that new method as `$this->crud->query()` which I think is more intuitive...